src/assets/arm_restraints/armbinder/armbinder.asset.ts

```diff
@@ -30,6 +30,11 @@ DefineAsset({
 		provides: [
 			'Restraint',
 			'Restraint_arms',
+			'Hand_item',
+			'Hand_cover',
+		],
+		covers: [
+			'Hand_item',
 		],
 	},
```

Comment:
Also, as an idea, since we thought about wanting to wear mittens under the armbinder: To make that possible in an RP without visual immersion breakage (for the steel ones), you could add a "hides: 'Mittens'," here if you want~